board/app/app_controller.php

```diff
@@ -11,14 +11,12 @@ function beforeFilter()
             'user/login'
         );
 
-        if (in_array(Param::get(DC_ACTION), $exclude)) 
-            {
-                return;
-            }
+    if (in_array(Param::get(DC_ACTION), $exclude)) {
+        return;
+    }
 
-         if (!is_logged_in()) 
-            {
-                redirect(url('user/login'));
-            }
+    if (!is_logged_in()) {
+        redirect(url('user/login'));
+        }
```

Comment:
done po. thank you :bow:
--> https://github.com/maragatchalian/FoodFacts/commit/95578cfbfa8e84aee41692a7d678cda9e6dd0823